tools/acrn-manager/README.rst

```diff
@@ -41,17 +41,21 @@ Add a VM
 ========
 
 The ``add`` command lets you add a VM by specifying a
-script that will launch a UOS, for example ``launch_UOS.sh``:
+script that will launch a UOS, for example ``launch_uos.sh``:
 
 .. code-block:: none
 
-   # acrnctl add launch_UOS.sh -U 1
+   # acrnctl add launch_uos.sh -U 1
    vm1-14:59:30 added
 
 If a ``-C`` option is also specified, the VM is launched in a runC
 container::
 
-   # acrnctl add launch_UOS.sh -C
+   # acrnctl add launch_uos.sh -C
+
+.. note:: You can get the `example
```

Comment:
Same change here:
```
.. note:: You can download an `example launch_uos.sh script 
   <https://raw.githubusercontent.com/projectacrn/acrnhypervisor/master/devicemodel/samples/nuc/launch_uos.sh>`_
   that supports the ``-C``  (``run_container`` function) option.
```

@gvancuts was hoping you'd add more information about how the ``-C`` option is handled (for example, all the work is done in the ``launch_uos.sh`` script and not by the ``acrnctl`` command) and more about what running in a runC container means.

Reply:
Hi Dave,
   The "-C" parameter is handled by launch_uos.sh script,  but every time you add the VM by acrnctl add command you need tell acrnctl this VM need launched with "-C" parameter, so acrnctl need store the "-C" parameter of the VM.
   If a VM is added with "-C" parameter, every time you start the VM, a runC container will be started at first, then the acrn-dm will run in the container. You can get this message from the ACRN-dm QoS HLD diagram.

Thanks

Reply:
> The "-C" parameter is handled by launch_uos.sh script, but every time you add the VM by acrnctl add command you need tell acrnctl this VM need launched with "-C" parameter, so acrnctl need store the "-C" parameter of the VM.

Unless I missed something in the `acrnctl` code, all `acrnctl` really does is check if `runc` is present on the system. It then strips `-C` from the current list of arguments and adds it back so the `launch_uos.sh` script picks it up. But doesn't `acrnctl` preserve all arguments that are placed after the launch script anyway?

(And checking if `runc` is available on the system could very easily be done in the `launch_uos.sh` script itself (e.g. `if [ -x /sbin/runc]`)

> If a VM is added with "-C" parameter, every time you start the VM, a runC container will be started at first, then the acrn-dm will run in the container. You can get this message from the ACRN-dm QoS HLD diagram.

How about we add a link to that doc from here?

Reply:
Hi Geoffroy,
   We have updated a new version with the QoS link in here, Rui is working on it.
Thanks